src/pages/CountryPage/CountryPage.tsx

```diff
@@ -30,29 +23,32 @@ export const CountryPage = () => {
           <h1 className='country-name'>{name?.common}</h1>
 
           <div className="container-country-props">
-            <strong>Native Name: <p className='country-value'>{nativeName?.common}</p> </strong>
+            <strong>Native Name: <p className='country-value'>{(nativeName?.common) ? nativeName?.common : "N/A"}</p> </strong>
             <strong>Population: <p className='country-value'>{population}</p> </strong>
             <strong>Region <p className='country-value'>{region}</p> </strong>
-            <strong>Sub Region: <p className='country-value'>{subregion}</p> </strong>
-            <strong>Capital: <p className='country-value'>{capital}</p> </strong>
+            <strong>Sub Region: <p className='country-value'>{(subregion) ? subregion : "N/A"}</p> </strong>
+            <strong>Capital: <p className='country-value'>{(capital) ? capital : "N/A"}</p> </strong>
           </div>
 
           <div className="container-country-props">
             <strong>Top Level Domain: <p className='country-value'>{tld}</p> </strong>
-            <strong>Currencies: <p className='country-value'>{currencyName.map(currency => currency).join(', ')}</p> </strong>
-            <strong>Languages: <p className='country-value'>{languageTypes.map(language => language).join(', ')}</p> </strong>
+            <strong>Currencies: <p className='country-value'>
+              {(currencyName && currencyName.length > 0) ? currencyName.map(currency => currency).join(', ') : "N/A"} 
+              </p>
+            </strong>
+            <strong>Languages: <p className='country-value'>
+              {(languagesName && currencyName.length > 0) ? languagesName.map(language => language).sort().join(', ') : "N/A"} </p>
```

Comment:
Creo que el `.map` está de más, salvo que esté equivocado:

```suggestion
              {(languagesName && currencyName.length > 0) ? languagesName.sort().join(', ') : "N/A"} </p>
```

Adicionalmente, si se piensa convertir ese array en un string, lo ideal es que este dato se manipule dentro del hook para que te lo retorne ya con el formato deseado (a no ser que el `languagesName` se utilice en otras partes)